### ISSUE TITLE
Enhancement Idea: aSPICE Pro - Allow ProxmoxVE connection with no .vv file

### ISSUE THREAD

**Issue by OptimusPi** [iiordanov/remote-desktop-clients#415]
Currently, aSPICE Pro works GREAT for my Proxmox VM. 

Here is how I connect to my VM from my iPad: 

But, it's inconvenient to first open Proxmox web interface, click "console" -> "spice", and let the .vv file download. 
Then I have to click the downloads and click Unknown.vv and it will open in aSPICE Pro

Here is how I want it to work: 

A way to save to a config so I don't have to type it every time would be great! I realize this is a big ask because of Proxmox ticketing authentication system -- the .vv file is only good for a few moments before it expires. 

Here is some information to perhaps help get this done:
https://github.com/Corsinvest/awesome-proxmox-ve
https://github.com/Corsinvest/cv4pve-pepper

**Comment by iiordanov** [comment]
Yes, it's quite a bit of work to get that done, but it is a valid request.
